api/meta/build/meta.yaml

```diff
@@ -350,7 +350,7 @@ components:
                   type: object
                   properties:
                     gps:
-                      $ref: '#/paths/~1return_reasons/post/requestBody/content/application~1json/schema/properties/context/allOf/0/properties/location/allOf/0/properties/gps/allOf/0'
+                      $ref: '/paths/~1return_reasons/post/requestBody/content/application~1json/schema/properties/context/allOf/0/properties/location/allOf/0/properties/gps/allOf/0'
```

Comment:
Can you please, specify the mechanic behind this fix? From what I found the part with the `#` was correct. As specified in docs:
The value of $ref uses the [JSON Reference](https://tools.ietf.org/html/draft-pbryan-zyp-json-ref-03) notation, and the portion starting with # uses the [JSON Pointer](https://tools.ietf.org/html/rfc6901) notation. This notation lets you specify the target file or a specific part of a file you want to reference. In the previous example, `#/components/schemas/User` means the resolving starts from the root of the current document, and then finds the values of `components`, `schemas`, and `User` one after another.
https://swagger.io/docs/specification/using-ref/

Reply:
You are right @olegromashin. The notation was correct all along. However, the $ref path itself looks a bit weird,

```
#/paths/~1return_reasons/post/requestBody/content/application~1json/schema/properties/context/allOf/0/properties/location/allOf/0/properties/gps/allOf/0
```

It should simply be `$ref : '#/components/schemas/Gps.yaml'` with the Gps schema inside the `components/schemas`

In any case, closing this issue and creating a new one related to the `$ref` error.